Design/Mantid_As_a _SciPy_Styled_Package.md

```diff
@@ -0,0 +1,259 @@
+Motivation
+==========
+
+Shaping Mantid as a SciPy styled package is one of the strategic aims in the [ORNL Mantid 5 year plan](ORNL_Mantid_5yearplan.pdf).
+However it has become clear over several meetings and workshops that what should be included as part of this aim is not commonly agreed or 
+understood within the development team or management of the project.
+This document is an attempt to harmonise this understanding or at least provide a center point for discussion and agreement or what we want 
+to achieve as part of this strategic aim and to provide a baseline high level design of what we need to do to implement it.
+
+Requirements
+============
+
+As mentioned above the requirements underlying the strategic aim are not universally shared so this document will serve as a place to detail
+what we have agreed.
+
+This is an initial draft from a discussion between Nick Draper and Martyn Gigg and will need to be discussed and agreed with 
+other team leads.
+
+Must
+----
+
+1. The Algorithms need to be separated into sensible sub packages that allow sensible imports of only the areas of interest 
```

Comment:
To be clear, you're talking about python modules here for the ease of python import? To distinguish this from a code directory reorganisation (although it might involve that) or includes at the c++ level.